metadata-io/src/main/java/com/linkedin/metadata/timeseries/elastic/indexbuilder/MappingsBuilder.java

```diff
@@ -34,6 +35,7 @@ public static Map<String, Object> getMappings(@Nonnull final AspectSpec aspectSp
 
     Map<String, Object> mappings = new HashMap<>();
 
+    mappings.put(RUN_ID_FIELD, ImmutableMap.of("type", "keyword"));
```

Comment:
what is the impact of adding this index?

Reply:
This allows for efficiently performing the rollback operation on time-series indices by allowing us to specify the `runId` field as part of the filter in a delete-by-query operation. There would be a slight increase in the index size. Also, there is no impact on the existing index data.